.github/workflows/cd.yaml

```diff
@@ -18,7 +18,7 @@ jobs:
         with:
           python-version: "3.8"
       - name: Build & Publish to PyPi
-        uses: JRubics/poetry-publish@v2.0
-        with:
-          pypi_token: ${{ secrets.PYPI_TOKEN }}
-          poetry_publish_options: "--build"
+        run: |
+          pip install poetry
+          poetry publish --build --username="__token__" --password="${{ secrets.PYPI_TOKEN }}"
```

Comment:
What does it mean that the username is `__token__`?

Reply:
You can either use password and username for authentication or an API token. If you use the token you have to set the username to `__token__`. This is described on [their help page](https://pypi.org/help/#apitoken)